src/Microsoft.AspNet.Mvc.Core/Logging/ActionDescriptorValues.cs

```diff
@@ -21,6 +21,7 @@ public ActionDescriptorValues([NotNull] ActionDescriptor inner)
             Name = inner.Name;
             DisplayName = inner.DisplayName;
             Parameters = inner.Parameters.Select(p => new ParameterDescriptorValues(p)).ToList();
+            CommonParameters = inner.CommonParameters.Select(p => new ParameterDescriptorValues(p)).ToList();
```

Comment:
No need to add this IMO. it doesn't hurt but all this code is going to be ripped up and done over again soon.